README.md

```diff
@@ -21,12 +21,12 @@ It is also possible to install this package as a global composer dependency.
 In order to add `phpcs:ignore` and `phpcs:disable`/`phpcs:enable` instructions throughout your project, run:
 
 ```sh
-vendor/bin/phpcs-baseliner create-baseline
+./vendor/bin/phpcs-baseliner create-baseline
```

Comment:
This shouldn't be necessary. `vendor/bin/phpcs-baseliner` and `./vendor/bin/phpcs-baseliner` are both relative paths, so this does not make any difference.